components/tabs/tab-mixin.js

```diff
@@ -112,6 +112,12 @@ export const TabMixin = superclass => class extends SkeletonMixin(superclass) {
 		}
 	}
 
+	dispatchChangeEvent() {
```

Comment:
I'm not sure how much Daylight documentation we're going to provide for making custom tab handles. Whether in Daylight docs, here in the mixin, or in `d2l-tab` as an example (or maybe all three), we'll definitely want to mention something to the effect of... "so that tabs can resize/manage its virtual scrolling" so that anyone writing a custom tab knows when/why they need to call this.

Reply:
I was picturing a section in the Daylight docs but I can also add comments to the other mentioned places.